neurom/geom/tests/test_transform.py

```diff
@@ -28,14 +28,30 @@
 
 import neurom.geom.transform as gtr
 from neurom.core.dataformat import COLS
-from neurom.core.tree import val_iter, ipreorder
+from neurom.core.tree import val_iter, ipreorder, Tree, make_copy
+from neurom.core.neuron import Neuron, make_soma
 from neurom.ezy import load_neuron
 from nose import tools as nt
 from itertools import izip
 import numpy as np
+from copy import copy
 
+TREE = Tree(np.array([0.0, 0.0, 0.0, 1.0, 1, 1, 2]))
```

Comment:
What do you mean by sliceable? lists and tuples are sliceable by pythons definition of that term. numpy. Anyway, there is no requirement that the neuron points be stored as numpy arrays. So if this doesn't work with lists and tuples, we have to re-think it.